code/processes/gateway.q

```diff
@@ -535,6 +535,18 @@ addserversfromconnectiontable[.servers.CONNECTIONS]
 // Join active .gw.servers to .servers.SERVERS table
 activeservers:{lj[select from .gw.servers where active;`handle xcol `w xkey .servers.SERVERS]}
 
+/function to tell orchestrator to scale up
```

Comment:
The code here is duplicated across both.  Could you have a generic function to cover both scenarios and either use params or projections?

Reply:
We were still working on adjusting the gateway code with differences between scaling up and down and this made it simpler. It can be a generic function with these changes added, I'll give it a look.